lib/txdb/handlers/triggers/handler.rb

```diff
@@ -26,7 +26,7 @@ def handle_safely
 
         def database
           @database ||= Txdb::Config.databases.find do |database|
-            database.database == request.params['database']
+            database.name == request.params['database']
```

Comment:
Well it can, but it's kind of awkward for the caller to specify a name, since name is pretty generic. The url they're hitting looks something like `blahblah.com/pull.json?database=abc&table=foo`. So we could change the params to database_name and table_name respectively, but just name isn't specific enough.